static/src/javascripts/projects/common/modules/experiments/tests/giraffe.js

```diff
@@ -0,0 +1,63 @@
+define([
+    'common/utils/$',
+    'common/utils/template',
+    'text!common/views/giraffe-message.html',
+    'common/views/svgs'
+], function ($,template,giraffeMessage, svgs) {
+    return function () {
+
+        this.id = 'Giraffe';
+        this.start = '2016-07-03';
+        this.expiry = '2016-08-13';
+        this.author = 'Alex Ware';
+        this.description = 'Add a button allowing readers to contribute money.';
+        this.showForSensitive = false;
+        this.audience = 0.001;
```

Comment:
I'm not sure 0.1% of the audience is enough to give you statistical significance across 4 variants. Might want to check with someone on the MVT team what you would need to run it on (I would run it minimum 4% to give each variant 1% personally but I don't know the numbers)